cimi-resources/test/com/sixsq/slipstream/ssclj/resources/spec/evidence_test.cljc

```diff
@@ -0,0 +1,56 @@
+(ns com.sixsq.slipstream.ssclj.resources.spec.evidence-test
+  (:require
+    [clojure.spec.alpha :as s]
+    [clojure.test :refer :all]
+    [com.sixsq.slipstream.ssclj.resources.evidence :as ev]))
+
+(def valid-acl {:owner {:principal "ADMIN"
+                        :type      "ROLE"}
+                :rules [{:principal "USER"
+                         :type      "ROLE"
+                         :right     "VIEW"}]})
+
+(def timestamp "1964-08-25T10:00:00.0Z")
+
+(def evid {:id            (str ev/resource-url "/uuid")
+                  :resourceURI   ev/resource-uri
+                  :created       timestamp
+                  :updated       timestamp
+                  :acl           valid-acl
+
+                  :name          "name"
+                  :description   "short description",
+                  :properties    {:a "one",
+                                  :b "two"}
+
+                  ; :credentials   [{:href "credential/e3db10f4-ad81-4b3e-8c04-4994450da9e3"}]
+
+                  :endTime        timestamp
+                  :startTime      timestamp
+                  :planID         "b12345"
+                  :passed         true
+                  :class          "className"
+
+                  :className:random 3147.12
+
+                  :log            ["log1","log2"]
+
+                  :other         "value"})
+
+
+(deftest check-ServiceInfo
+
+  (are [expect-fn arg] (expect-fn (s/valid? :cimi/evidence arg))
```

Comment:
This is probably more concise and clear by using the `doseq` pattern that you can find in, for example, the `external_object_test.cljc`.

Reply:
done